internal/fsapi/file.go

```diff
@@ -380,3 +375,59 @@ type File interface {
 	//     https://pubs.opengroup.org/onlinepubs/9699919799/functions/close.html
 	Close() syscall.Errno
 }
+
+// Readdir is the status of a prior fs.ReadDirFile call.
```

Comment:
Seems this is out-of date? My thought would be that this is the result of readdir, not a status, which is effectively a cursor over a directory.